src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+      int color = new Random().nextInt(Color.values().length);
+      return Color.values()[color];
     }
```

Comment:
Creating a new instance of `Random` every time the method is called can be inefficient. Consider creating a single instance of `Random` as a class field and reusing it. Additionally, the `Color.values()` method creates a new array each time it is called, which can be avoided by storing the array in a constant field.